hr_employee_medical_examination/views/hr_employee_medical_examination_views.xml

```diff
@@ -113,13 +132,62 @@
+        <field name="name">hr.employee.medical.examination.kanban</field>
+        <field name="model">hr.employee.medical.examination</field>
+        <field name="arch" type="xml">
+            <kanban>
+                <field name="state" />
+                <field name="id" />
+                <field name="employee_id" />
+                <field name="date" />
+                <field name="activity_ids" />
+                <templates>
+                    <t t-name="kanban-box">
+                        <div class="oe_kanban_global_click o_hr_kanban_record">
+                            <div class="oe_kanban_details">
+                                <div class="o_kanban_record_top">
+                                    <div class="o_kanban_record_headings">
+                                        <strong class="o_kanban_record_title">
+                                            <field name="name" />
+                                        </strong>
+                                        <span class="o_kanban_record_subtitle"><field
+                                                name="employee_id"
+                                            /></span>
+                                    </div>
+                                </div>
+                                <div class="o_kanban_record_bottom">
+                                    <div class="oe_kanban_bottom_left text-muted">
+                                        <span>
+                                            <t t-out="record.date.value" />
+                                        </span>
+                                        <field
+                                            name="activity_ids"
+                                            widget="kanban_activity"
+                                        />
+                                    </div>
+                                    <div class="oe_kanban_bottom_right">
+                                        <field
+                                            name="state"
+                                            widget="label_selection"
+                                            options="{'classes': {'pending': 'default', 'cancelled': 'default', 'rejected': 'default', 'done': 'success'}}"
+                                        />
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </t>
+                </templates>
+            </kanban>
+        </field>
+    </record>
     <record
         model="ir.actions.act_window"
         id="hr_employee_medical_examination_act_window"
     >
         <field name="name">Employee Medical Examinations</field>
         <field name="res_model">hr.employee.medical.examination</field>
-        <field name="view_mode">tree,form</field>
+        <field name="view_mode">kanban,tree,form</field>
```

Comment:
Why `kanban` is before `tree` one?

Reply:
Initially, I thought that the kanban view was more useful, which is why I defined it first, but your question made me think that it might cause more confusion than improvement, so I changed it so that it is not defined first.